### ISSUE TITLE
Receiver status unknown in list but displayed as down in detail page

### ISSUE THREAD

**Issue by pyrog** [kerel-fs/ogn-rdb#10]
As the APRS is really unknown (glidern1 aprsc server is down today), the list display no state.
<img width="1158" alt="List" src="https://cloud.githubusercontent.com/assets/2158081/20638666/cadd121a-b3ad-11e6-8755-c7665b5d8d27.png">

But the detail page display Down
<img width="643" alt="Detail" src="https://cloud.githubusercontent.com/assets/2158081/20638665/cab70570-b3ad-11e6-8408-443ffe57cc98.png">

PS: screen copies taken from my local repository.

**Comment by pyrog** [comment]
@kerel-fs You could add theses lines to `/etc/hosts` to reproduce the DNS issue / APRSC servers not reachable

```
# test OGN
192.168.1.31   glidern1.glidernet.org
192.168.1.32   glidern2.glidernet.org
192.168.1.33   glidern3.glidernet.org
```
**PS:** I used an local IP adress where I know that no server could respond.
(And don't forget to remove after the test 😉. I do that 😆)

**Comment by kerel-fs** [comment]
Fixed in 7539a36.
